src/main/java/org/broadinstitute/consent/http/service/DarCollectionService.java

```diff
@@ -159,6 +159,12 @@ private void processDarCollectionSummariesForResearcher(List<DarCollectionSummar
       int electionCount = elections.values().size();
       elections.values().forEach(election -> updateStatusCount(statusCount, election.getStatus()));
       s.addAction(DarCollectionActions.REVIEW.getValue());
+      // if any DARs in the collection have approved datasets, include create progress report action
+      if (s.getReferenceIds().stream()
+          .map(dataAccessRequestDAO::findApprovedDatasetsByDar)
```

Comment:
This will work for now, but it is a looped DAO call inside of another loop of summaries and will become problematic over time when a user has lots of DARs with many datasets. It would be more efficient to query for the datasets before the loop. This might require an additional query that does the same work as `dataAccessRequestDAO::findApprovedDatasetsByDar` but instead for a list of reference ids.